lib/fpm/package/rpm.rb

```diff
@@ -163,11 +163,15 @@ class FPM::Package::RPM < FPM::Package
   # Replace ? with [?] to make rpm not use globs
   # Replace % with [%] to make rpm not expand macros
   def rpm_fix_name(name)
-    name = "\"#{name}\"" if name[/\s/]
-    name = name.gsub("[", "[\\[]")
-    name = name.gsub("*", "[*]")
-    name = name.gsub("?", "[?]")
-    name = name.gsub("%", "[%]")
+    name = name.gsub(/(\ |\[|\]|\*|\?|\%|\$)/, {
+      ' ' => '?',
```

Comment:
This isn't correct, I don't think. Two files: 'foo bar' and 'foo.bar' will be matched by the result of this.

I'm not really sure what the right solution here is given rpmbuild's bugs with quoted strings and wildcards.